bsp/boards/iot-lab_M3/uart.c

```diff
@@ -95,12 +95,9 @@ void uart_clearTxInterrupts(void) {
 }
 
 void uart_setCTS(bool state) {
-    
-    if (state==0x01) {
-        USART_SendData(USART1,(uint16_t)XON);
-    } else {
-        USART_SendData(USART1,(uint16_t)XOFF);
-    }
+    // do nothing since the serial connection is bridged by a TCP

```

Comment:
With this changes, the M3 won't be able to work locally. Is there a problem if using the current implementation on FIT/IoT-LAB?

Reply:
XON/XOFF will be transmitted to user's terminal if I understand it correctly. If you want to use M3 locally, disabling the software flow control is one of solutions... :-/

Do we have a compilation option to disable XON/XOFF? ;-)

Reply:
Thank you for the review; I thought I had a wrong idea. I confirmed that the mercator firmware works without this change. I will revert this.

Reply:
Thanks so much for taking the comments!